src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/InteropServices/PlatformCompatibilityAnalyzer.cs

```diff
@@ -691,6 +715,23 @@ private static ISymbol GetEventAccessor(IEventSymbol iEvent, IOperation operatio
             else
             {
                 CheckOperationAttributes(operation, context, platformSpecificOperations, platformSpecificMembers, msBuildPlatforms, symbol);
+                if (symbol is IMethodSymbol method)
+                {
+                    if (method.IsGenericMethod)
+                    {
+                        foreach (var typeArgument in method.TypeArguments)
+                        {
+                            CheckOperationAttributes(operation, context, platformSpecificOperations, platformSpecificMembers, msBuildPlatforms, typeArgument);
+                        }
+                    }
+                    else if (method.ReceiverType is INamedTypeSymbol namedType && namedType.IsGenericType)
```

Comment:
`method.IsGenericMethod` could be used for generic method invocation, but for a constructor with a generic type parameter `method.IsGenericMethod` is somehow false and the `method` has no any info about the generic type, `method.ReceiverType` has the generic constructor info we need